enterprise/server/remote_execution/commandutil/commandutil.go

```diff
@@ -98,11 +102,15 @@ func RetryIfTextFileBusy(fn func() error) error {
 	}
 }
 
-// Run a command, retrying "text file busy" errors and killing the process group
+// Run a command, retrying "text file busy" errors and killing the process tree
 // when the context is cancelled.
-func Run(ctx context.Context, command *repb.Command, workDir string, stdio *container.Stdio) *interfaces.CommandResult {
+//
+// Note: enableStats incurs some overhead throughout process execution, so only
+// use it if stats are needed.
+func Run(ctx context.Context, command *repb.Command, workDir string, enableStats bool, stdio *container.Stdio) *interfaces.CommandResult {
```

Comment:
@tylerwilliams I wound up not creating the "simple" version of this func because there were only 1 or 2 callsites (excluding `commandutil_test.go`) that would've been able to take advantage of it. I figured we can revisit if there are more callsites and the need for a simpler API becomes more clear.